boltdesignsystem/bolt
# DS-177: add new brand colors

## Jira

https://pegadigitalit.atlassian.net/browse/DS-177

## Summary

Adds secondary and tertiary brand colors to the color system.

## Details

1. Added wine, berry, pink, and violet to the color map.
1. Removed xdark and xlight shades from orange and yellow.
1. Generated CSS custom props for all colors.
1. Updated color docs and added color shade toggle.

## How to test

Run the branch locally and check the color docs under Visual Styles. Create a dummy page and try out the new CSS custom props.

## Release notes

### Deprecated functions

- bolt-color(orange, xdark)
- bolt-color(orange, xlight)
- bolt-color(yellow, xdark)
- bolt-color(yellow, xlight)
